apps/ui/components/application/ApprovedReservations.tsx

```diff
@@ -514,6 +527,8 @@ function ReservationSeriesTable({
               accessTypes={modal.accessTypes}
               reservationUnits={reservationUnits}
               pk={modal?.pk}
+              seasonBeginDate={applicationRound.reservationPeriodBegin}
+              seasonEndDate={applicationRound.reservationPeriodEnd}
```

Comment:
Prefer similar prop name as what's it supposed to be given (This could be misinterpreted as **application period**, **reservation period** or **public display**)
e.g. `roundReservationBegin`